Makefile

```diff
@@ -38,7 +39,7 @@ $(LIBSGE):
 	$(MAKE) -C $(SGE)
 
 $(GAME): $(LIBSGE) $(OBJS) | spritesheet.png
-	$(CXX) -o $@ $^ $(GLFWDEPS) $(LDFLAGS)
+	$(CXX) -o $@ $^ $(GLFWDEPS) $(LDFLAGS) $(LIBSGE)
```

Comment:
`$(LIBSGE)` is already linked, because I'm using `$^` which includes all dependencies before the `|`.